langserver/references.go

```diff
@@ -31,7 +31,10 @@ func (h *LangHandler) handleTextDocumentReferences(ctx context.Context, conn JSO
 	}
 
 	bctx := h.OverlayBuildContext(ctx, h.defaultBuildContext(), !h.init.NoOSFileSystemAccess)
-	_, rev, _ := importgraph.Build(bctx)
+	h.importGraphOnce.Do(func() {
+		_, rev, _ := importgraph.Build(bctx)
+		h.importGraph = rev
```

Comment:
We ignored the errors previously. I assume we ignore the error since we are doing best effort, so we just ignore code which doesn't compile. I'll add in a comment.